secret_hitler.py

```diff
@@ -334,7 +334,14 @@ def show_logs(self, include_knowledge_of=None):
     @staticmethod
     def format_time(seconds):
         gmtime = time.gmtime(seconds)
-        return "{0:0>2}h {1:0>2}m".format(gmtime.tm_hour, gmtime.tm_min)
+        if gmtime.tm_yday > 1000:
```

Comment:
Note that this will never happen: `tm_yday` is the day of the year and therefore never larger than 366. That's a side effect of interpreting the *duration* as an *epoch time* :P

Reply:
See https://docs.python.org/3/library/time.html#time.struct_time

Reply:
Resolved?

Reply:
Apart from leap years that LGTM :new_moon_with_face: